publisher-provider/publisher-provider-app/src/main/java/nl/idgis/publisher/provider/sde/SDEGatherDatasetInfo.java

```diff
@@ -76,28 +74,49 @@
 	private String categoryId;
 	
 	private String databaseScheme;
-	
+
+	private String metadataTable;
+
 	private DatabaseTableInfo databaseTableInfo;
 
-	private Path rasterFile;
-	
 	private Set<Attachment> attachments;
 	
 	private ZonedDateTime revisionDate;
 	
 	private Map<String, String> attributeAliases;
 
-	private Config databaseConfig;
-
 	private Config rasterConfig;
-	
+
+	private final DatabaseType databaseVendor;
+
 	public SDEGatherDatasetInfo(ActorRef target, ActorRef transaction, ActorRef rasterFolder, Set<AttachmentType> attachmentTypes, Config databaseConfig, Config rasterConfig) {
 		this.target = target;
 		this.transaction = transaction;
 		this.rasterFolder = rasterFolder;
 		this.attachmentTypes = attachmentTypes;
-		this.databaseConfig = databaseConfig;
 		this.rasterConfig = rasterConfig;
+
+		try {
+			this.databaseScheme = databaseConfig.getString("scheme");
+		} catch(ConfigException.Missing cem) {
+			this.databaseScheme = "SDE";
+		}
+
+		if (databaseConfig.hasPath("vendor")) {
+			try {
+				this.databaseVendor = DatabaseType.valueOf(databaseConfig.getString("vendor").toUpperCase());
+				if (databaseVendor == DatabaseType.POSTGRES) {

```

Comment:
Overweeg om dit soort informatie in de DatabaseType enum op te nemen.

Reply:
Dit gaat om speciefieke SDE informatie. Hoort niet in een DatabaseType ENUM